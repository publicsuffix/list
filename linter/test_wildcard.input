// test:
// - valid wildcard usage
// - invalid wildcard usage

// ===BEGIN ICANN DOMAINS===

// valid
*.com

// invalid
**.com
a*.com
b.*.com
a.b.*

// ===END ICANN DOMAINS===
