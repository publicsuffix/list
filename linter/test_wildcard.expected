11: error: Illegal character: '**.com'
12: error: Illegal character: 'a*.com'
13: error: Illegal character: 'b.*.com'
14: error: Illegal character: 'a.b.*'
16: warning: No PRIVATE section found
